src/opensynth/datasets/low_carbon_london/preprocess_lcl.py

```diff
@@ -222,20 +275,27 @@ def create_outliers(
         pd.DataFrame: Dataframe consisting of noisy outliers
     """
 
+    if time_resolution == "half_hourly":
+        n = 48
+    elif time_resolution == "hourly":
+        n = 24
+    else:
+        raise ValueError("time_resolution must be 'half_hourly' or 'hourly'")
```

Comment:
Slightly more informative error message
```suggestion
        raise ValueError(f"time_resolution must be 'half_hourly' or 'hourly', got {time_resolution}")
```